fltkhs.cabal

```diff
@@ -12,29 +12,62 @@ bug-reports: https://github.com/deech/fltkhs/issues
 copyright: © 2017 Aditya Siram All Rights Reserved
 category: UI,Graphics,GUI,User Interfaces
 build-type: Custom
-cabal-version: >=1.24
+cabal-version: >=2.0
```

Comment:
Doesn't this need to stay `cabal-version: >=1.24` for those who are still on older versions?